src/components/layouts/AdminLayout.tsx

```diff
@@ -0,0 +1,83 @@
+import {
+  Drawer,
+  List,
+  ListItemButton,
+  ListItemIcon,
+  ListItemText,
+} from "@mui/material";
+import InboxIcon from "@mui/icons-material/MoveToInbox";
+import MailIcon from "@mui/icons-material/Mail";
+import { useRouter } from "next/router";
+import { type Route } from "nextjs-routes";
+
+const DRAWER_WIDTH = 240;
+
+type MenuItem = {
+  text: string;
+  icon: JSX.Element;
+  path: "/" | Route;
+};
+
+export default function AdminLayout({
+  children,
+}: {
+  children: React.ReactNode;
+}) {
+  const router = useRouter();
+
+  const menuItems: MenuItem[] = [
+    {
+      text: "Collections",
+      icon: <InboxIcon />,
+      path: "/",
+    },
+    {
+      text: "Forms",
+      icon: <MailIcon />,
+      path: "/",
+    },
+  ];
+
+  const handleNavigation = async (item: MenuItem) => {
+    const path = item.path;
+    await router.push(path);
+  };
+
+  return (
+    <div style={{ display: "flex" }}>
```

Comment:
There is nothing wrong with this but could we use Tailwind when possible? CSS in JS also has some performance downsides and by using Tailwind we avoid this potential slowdown and make the code consistent everywhere.